guides/micronaut-microservices-services-discover-eureka/bookinventory/kotlin/src/main/kotlin/example/micronaut/BookInventory.kt

```diff
@@ -1,3 +1,10 @@
 package example.micronaut
 
-data class BookInventory(var isbn: String, val stock: Int)
\ No newline at end of file
+import io.micronaut.core.annotation.Creator
+import io.micronaut.core.annotation.Introspected
+import javax.validation.constraints.NotBlank
+
+@Introspected
+data class BookInventory @Creator constructor(
```

Comment:
I don't think `@Creator` is necessary. Am I missing anything?

If you want to specify it to educate the user about `@Creator` I think we should add a callout with a callout snippet.